ci/docker/Dockerfile.build.ubuntu_nightly_gpu

```diff
@@ -74,6 +74,10 @@ ARG GROUP_ID=0
 COPY install/ubuntu_adduser.sh /work/
 RUN /work/ubuntu_adduser.sh
 
+# This needs to be run after /work/mxnet directory is created by ubuntu_adduser.sh.
```

Comment:
Sorry, I mean that this should be done in https://docs.python.org/2/library/unittest.html#unittest.TestCase.setUpClass

If you do it here, you have two problems:
1. The directory won't be valid since the docker build is independent from the runtime
2. The result will be cached and thus we might not be able to detect problems in the straight dope up front

I know that my suggestion is not ideal, but otherwise we got a test don't won't be runnable if executed without proper knowledge about dependencies.